x-pack/plugin/autoscaling/src/internalClusterTest/java/org/elasticsearch/xpack/autoscaling/action/TransportGetAutoscalingCapacityActionIT.java

```diff
@@ -14,17 +15,24 @@
 import org.hamcrest.Matchers;
 
 import java.util.Set;
+import java.util.StringTokenizer;
 import java.util.TreeMap;
 import java.util.TreeSet;
 
+import static org.apache.lucene.util.Constants.JVM_SPEC_VERSION;
 import static org.elasticsearch.test.hamcrest.ElasticsearchAssertions.assertAcked;
 import static org.hamcrest.Matchers.greaterThan;
 
 @ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 0)
 public class TransportGetAutoscalingCapacityActionIT extends AutoscalingIntegTestCase {
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/67089")
     public void testCurrentCapacity() throws Exception {
+        boolean looksLikeDebian8 = Constants.LINUX && Constants.OS_VERSION.startsWith("3.16.0");
+        final StringTokenizer st = new StringTokenizer(JVM_SPEC_VERSION, ".");
```

Comment:
We have `JavaVersion.CURRENT.compareTo(JavaVersion.parse("15"))`.

Reply:
Thanks, fixed in 2943006da10fb534e8c423bb72d2a35724588f5f